README.md

```diff
@@ -18,6 +18,13 @@ You can start editing the page by modifying `pages/index.js`. The page auto-upda
 
 The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
 
+## Additional Commands
+
+| Command   | Description |
+|-----------|-------------|
+| `npm run test` | runs a single test command. Get feedback one time |
+| `npm run test:watch` | constantly looks for changes to your files and reruns tests. Get continuous feedback (you can always ^C to exit) |
```

Comment:
The most important part of the documentation is your audience so check with your group, but this might be overly documented and is causing your description to be quite long. 

<img width="1061" alt="Screen Shot 2022-10-07 at 1 03 09 PM" src="https://user-images.githubusercontent.com/7015466/194611760-338550ad-0374-4dff-a3f7-15f91334fcf5.png">

**Three considerations:**
1. What does "Get feedback one time" communicate that "runs a single test command" does not?
2. What does "Get continuous feedback" communicate that "constantly looks for changes to your files and reruns tests" does not?
3. Control + C is not specific to this project but, instead, is the keyboard shortcut to kill _any_ running process. Do we think it's important enough to include here? Would we similarly add this reminder to a description to other scripts like `npm start` that start running processes?

More documentation isn't always better. In fact, in many cases, it can distract from what's really important. And, very minor documentation nitpick, parts of your description are in sentence case and some are in lowercase. I'd consistently use sentence case for descriptions.

Reply:
I was unhappy with how that table looked and now I see it's probably because I basically regurgitated my own notes.

Reply:
Nobody needs to see those!!

Reply:
[ae43b48](https://github.com/sparkbox/accessible-components/pull/17/commits/ae43b48d9dfade5b607cdf78d9a141a71ce1176d)